cmd/cloudflared/tunnel/subcommands.go

```diff
@@ -397,6 +398,30 @@ func fmtConnections(connections []cfapi.Connection, showRecentlyDisconnected boo
 	return strings.Join(output, ", ")
 }
 
+func buildHealthCommand() *cli.Command {
+	return &cli.Command{
+		Name:               "health",
+		Action:             cliutil.ConfiguredAction(healthCommand),
+		Usage:              "Tunnel health exit code",
+		UsageText:          "cloudflared tunnel [tunnel command options] health [subcommand options]",
+		Description:        "cloudflared tunnel health will return proper exit code if tunnel is healthy or unhealthy",
+		Flags:              []cli.Flag{},
+		CustomHelpTemplate: commandHelpTemplate(),
+	}
+}
+
+func healthCommand(c *cli.Context) error {
+	requestURL := fmt.Sprintf("http://%s/ready", c.String("metrics"))
```

Comment:
I think the `/healthcheck` endpoint would be more appropriate.

Reply:
I wasn't aware that such an endpoint existed :+1: I will change it. @felipelima-circle